default/buildings.txt

```diff
@@ -625,7 +625,7 @@ BuildingType
             stackinggroup = "INDUSTRY_CENTER_STACK"
             effects = SetTargetIndustry value = Value + Target.Population * 3 * [[INDUSTRY_PER_POP]]
     ]
-    icon = ""
+    icon = "icons/tech/industrial_centre_ii.png"
```

Comment:
Don't understand a) what exactly you mean there and more importantly b) how to do it, I've tried rebasing and squashing stuff, in this particular case I was testing Sloth's patch while working on the buildings and didn't realise Git would put them together the way it has, I know now but it's still the way it'd work out normally.